### ISSUE TITLE
bug in hidden state?

### ISSUE THREAD

**Issue by qiuqiangkong** [deepsound-project/samplernn-pytorch#32]
Hi, 

    Thanks for the good work! I am wondering should https://github.com/deepsound-project/samplernn-pytorch/blob/master/model.py#L49 be:

    h0 = torch.zeros(n_rnn, batch_size, dim)?

    I noticed the expand function is used later, but it seems using the expand function the values are shared. Is this a bug? 

Many thanks! 

Qiuqiang

**Comment by qiuqiangkong** [comment]
I found the .contiguous() function fixed this problem.
